src/PresentationalComponents/RuleRating/RuleRating.js

```diff
@@ -1,33 +1,25 @@
 import './_RuleRating.scss';
 
-import * as AppConstants from '../../AppConstants';
-
 import React, { useState } from 'react';
 
-import API from '../../Utilities/Api';
 import { Button } from '@patternfly/react-core/dist/js/components/Button/Button';
 import OutlinedThumbsDownIcon from '@patternfly/react-icons/dist/js/icons/outlined-thumbs-down-icon';
 import OutlinedThumbsUpIcon from '@patternfly/react-icons/dist/js/icons/outlined-thumbs-up-icon';
 import PropTypes from 'prop-types';
 import ThumbsDownIcon from '@patternfly/react-icons/dist/js/icons/thumbs-down-icon';
 import ThumbsUpIcon from '@patternfly/react-icons/dist/js/icons/thumbs-up-icon';
-import { injectIntl } from 'react-intl';
```

Comment:
Swap this for use intl n instantiate it in the component rather than passing it in